tools/topology/topology2/development/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+# SPDX-License-Identifier: BSD-3-Clause
+
+# Array of "input-file-name;output-file-name;comma separated pre-processor variables"
+set(TPLGS
+# CAVS SSP topology for MTL for electrical validation
+# Default MCLK = 24576000, BCLK = 3072000, FSYNC = 48000,
+# TDM_SLOTS = 2, TDM_WIDTH = 32
+# Verify with
+# MCLK = 38400000, BCLK = 4800000, FSYNC = 48000,
+# TDM_SLOTS = 4, TDM_WIDTH = 25
+"cavs-nocodec-val\;sof-mtl-nocodec-1\;PLATFORM=mtl,NUM_DMICS=2,\
+MCLK=38400000,BCLK=4800000,TDM_SLOTS=4,TDM_WIDTH=25,\
+PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-ace-mtl-nocodec-1.bin"
+# Verify with
+# MCLK = 19200000, BCLK = 6144000, FSYNC = 48000,
+# TDM_SLOTS = 4, TDM_WIDTH = 32
+"cavs-nocodec-val\;sof-mtl-nocodec-2\;PLATFORM=mtl,NUM_DMICS=2,\
+MCLK=19200000,BCLK=6144000,TDM_SLOTS=4,\
+PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-ace-mtl-nocodec-2.bin"
+# Verify with
+# MCLK = 19200000, BCLK = 12280000, FSYNC = 96000,
+# TDM_SLOTS = 4, TDM_WIDTH = 32
+"cavs-nocodec-val\;sof-mtl-nocodec-3\;PLATFORM=mtl,NUM_DMICS=2,\
+MCLK=19200000,BCLK=12280000,TDM_SLOTS=4,FSYNC=96000,\
+PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-ace-mtl-nocodec-3.bin"
+# Verify with
+# MCLK = 38400000, BCLK = 4800000, FSYNC = 96000,
+# TDM_SLOTS = 2, TDM_WIDTH = 25
+"cavs-nocodec-val\;sof-mtl-nocodec-4\;PLATFORM=mtl,NUM_DMICS=2,\
+MCLK=38400000,BCLK=4800000,TDM_WIDTH=25,FSYNC=96000,\
+PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-ace-mtl-nocodec-4.bin"
+# Verify with
+# MCLK = 19200000, BCLK = 6144000, FSYNC = 96000,
+# TDM_SLOTS = 4, TDM_WIDTH = 32
+"cavs-nocodec-val\;sof-mtl-nocodec-5\;PLATFORM=mtl,NUM_DMICS=2,\
+MCLK=19200000,BCLK=6144000,TDM_SLOTS=4,FSYNC=96000,\
+PREPROCESS_PLUGINS=nhlt,NHLT_BIN=nhlt-ace-mtl-nocodec-5.bin"
+)
+
```

Comment:
the output files seem to be named/differentiated by increasing integer number, it may might be good to name those on the parameter value you are vaying. just to make it clear/readable.

Reply:
renamed to include mclk, bclk and fsync in the filename